debian/nymea-mqtt-tests.install

```diff
@@ -0,0 +1 @@
+/usr/bin/nymeamqtttests*
```

Comment:
I know it's wrong in the nymea-tests package too, but:

"According to Debian policy, test files should be placed in the /usr/share/tests directory. Inside this directory, you can organize your test files further based on the software package they belong to. For example:

/usr/share/tests/<package-name>/
"

Reply:
good point, I will allign all the paths